bitcoin-dev-project/warnet
# Add Loki log aggregation

Compose only.

Adds log aggregation using Loki and Promtail and makes it available in Grafana.

![Screenshot 2024-01-04 at 14 14 28](https://github.com/bitcoin-dev-project/warnet/assets/1204616/b54be5f1-3d17-4f46-b5a6-72e6521f1baf)

Also moves services directory to within the backends and specifically the compose backend. Services will be backend specific.

Suggested steps to test:

1. Run warnet network on docker compose
2. Open http://localhost:3000/explore
3. Select Loki datasource
4. Use the following query to see logs from tanks: `{container=~".+tank.+"}`
5. Click Run Query
